tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.cs

```diff
@@ -51,9 +55,14 @@ private static void LogMatchesIfDebugEnabled(string? result, bool blocked)
         }
     }
 
-    public IResult? Scan()
+    public IResult? Scan(bool firstTime = false)
     {
         var args = GetBasicRequestArgsForWaf();
+        if (firstTime)
+        {
+            _security.ApiSecurity.TryTellWafToAnalyzeSchema(args);
```

Comment:
I love this XD `TryTellWafPleaseIfItFeelsLikeAnalyzingTheSchema`

Reply:
Yes, I want to infer the meaning of `Try` because otherwise the naming would give the impression it's doing it in all cases, whereas if it cant secure the request, it wont do it ^^